src/test/java/com/znsio/api/CreateAndEditThePost.feature

```diff
@@ -0,0 +1,22 @@
+@sampleApi @prod
+Feature: Create and edit post
+
+  Background: API url
+
+  Scenario: Create a post and fetch the created post
+    Given def createdPost = karate.call('classpath:com/znsio/templates/CreateAndEditThePostTemplates.feature@t_createAPost').createPost
+    And assert createdPost.id > 100
+    And match createdPost.id == '#number'
+    And match createdPost.userId == '#number'
+    * def Id = createdPost.id
+
+    Given def listOfPosts = karate.call('classpath:com/znsio/templates/CreateAndEditThePostTemplates.feature@t_fetchAPost').fetchPost
+    And match listOfPosts.id == Id
+    And match listOfPosts.userId == '#number'
+
+   Scenario: Update the created post title and verify the title
```

Comment:
Does this scenario expect a post to be already available? What will happen if it is not available?